### PR TITLE
Fix Handler.handle return type

### PR DESCRIPTION
**Pull request by aminalaee**
- The argument type to `Handler.emit` is not correct
- The return type of `Handler.handle` should be True or a falsy value (this is changed for Py 3.12 a bit but until then this is the correct one).

### CODE REVIEW

**Review by tonybaloney — APPROVED** [review]
Makes sense, thanks for the update

File: src/picologging/handler.cxx [hunk 1]
```diff
@@ -70,7 +70,13 @@ PyObject* Handler_handle(Handler *self, PyObject *record) {
     }
     
     self->lock->unlock();
-    return result;
+
+    // emit failed
```
Comment (tonybaloney) [inline src/picologging/handler.cxx:74]: You could write this as a ternary expression instead, I think something like:

```cpp
return result == nullptr ? nullptr : Py_True;
```

Comment (aminalaee) [reply src/picologging/handler.cxx]: Thanks, done.

**Review by tonybaloney — APPROVED** [review]

**Comment by tonybaloney** [comment]
Great stuff, thanks!

**Pull request merged**